app/templates/webpack.config.js

```diff
@@ -1,14 +1,22 @@
 const path = require('path');
 const webpack = require('webpack');
-const UglifyJsPlugin = webpack.optimize.UglifyJsPlugin;
 
 const libraryName = '<%= libraryNameCamelized %>';
+const isProduction = process.env.NODE_ENV === 'production';
+
+const UglifyJsPlugin = webpack.optimize.UglifyJsPlugin;
+const extractStyles = new ExtractTextPlugin({
+  filename: 'styles/[name].[contenthash].css',
+  allChunks: true,
+  // disable: !isProduction
+})
+const enableSourcemaps = true
 
 module.exports = {
   entry: path.join(__dirname, 'src/index.js'),
   output: {
-    path: path.join(__dirname, 'lib'),
-    filename: libraryName + '.min.js',
+    path: path.join(__dirname, 'dist'),
```

Comment:
Should this be `lib` instead of `dist`? Also, we can prob get rid of any references to `dist` in the project - I was using it as a way to test a prod build of the example app, but it's not really being used anymore.

Reply:
Yup, I believe you are correct. I believe this is from when I was trying out commonjs/es builds (had an issue getting styling working though).